.github/workflows/build_and_test.yml

```diff
@@ -41,6 +41,26 @@ jobs:
     with:
       module: save-demo-agent
       gradle-cache-read-only: ${{ github.ref != 'refs/heads/master' && github.event_name != 'pull_request' }}
+  retrieve_save_cli_version:
+    name: Retrieve save-cli version from save-cloud
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.save-save-cli-version.outputs.version }}
+    steps:
+      - name: checkout save-cloud
+        uses: actions/checkout@v3
+      - id: save-save-cli-version
+        name: Store save-cli version in output
+        run: |
+          sv=$(cat gradle/libs.versions.toml | grep -m1 '^save-cli =' | awk -F'[=]' '{print $2}' | tr -d '" ')
+          echo "Detected version $sv"
+          echo version=$sv >> $GITHUB_OUTPUT
+  build_cli:
+    name: Build save-cli
+    needs: [ retrieve_save_cli_version ]
+    uses: ./.github/workflows/build_save-cli_reusable.yml
```

Comment:
do we really need the first `.`?

Reply:
taken from [documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow): 
```
You reference reusable workflow files using one of the following syntaxes:

- {owner}/{repo}/.github/workflows/{filename}@{ref} for reusable workflows in public and private repositories.
- ./.github/workflows/{filename} for reusable workflows in the same repository
```

But, I didn't test without leading `.`